### PR TITLE
Update to use addon service API for addons function

### PR DESCRIPTION
**Pull request by weherdh**
This is to resolve OCM-1983, to update the cli to list addons with ocm-addon-service API

### CODE REVIEW

**Comment by ashishmax31** [comment]
This looks good. @ciaranRoche Are there any other places where we need to switch over the API?

File: go.mod [hunk 1]
```diff
@@ -114,3 +114,5 @@ require (
 	google.golang.org/protobuf v1.34.0 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 )
+
+replace github.com/openshift-online/ocm-sdk-go => github.com/venkateshsredhat/ocm-sdk-go v0.1.445-newtest
```
Comment (venkateshsredhat) [inline go.mod:118]: Not required , use the latest https://github.com/openshift-online/ocm-sdk-go/releases/tag/v0.1.447

File: pkg/cluster/cluster.go [hunk 2]
```diff
@@ -767,7 +774,14 @@ func GetClusterAddOns(connection *sdk.Connection, clusterID string) ([]*AddOnIte
...
 
 	// Get add-ons already installed on cluster
-	addOnInstallationsResponse, err := connection.ClustersMgmt().V1().Clusters().
+	//addOnInstallationsResponse, err := connection.ClustersMgmt().V1().Clusters().
+	//	Cluster(clusterID).
+	//	Addons().
+	//	List().
+	//	Page(1).
+	//	Size(-1).
+	//	Send()
```
Comment (ciaranRoche) [inline pkg/cluster/cluster.go:783]: All the commented out code can be removed.

**Review by ciaranRoche — COMMENTED** [review]
You will also need to update the addons in pkg/url_expander.go

**Comment by weherdh** [comment]
> You will also need to update the addons in pkg/url_expander.go

Updated

**Review by ciaranRoche — APPROVED** [review]
lgtm

**Pull request merged**